src/assets/locales/fr.json

```diff
@@ -519,6 +589,17 @@
   "tips.mobile.locomotion": "Bravo ! Pour vous déplacer, pincez avec deux doigts.",
   "tips.mobile.look": "Bienvenue ! 👋 Cliquez et faites glisser pour regarder autour de vous.",
   "tips.streaming": "Diffusion en cours vers le hall d'attente. Pour quitter le mode diffusion, utilisez le menu Plus quand vous aurez terminé.",
+  "token-type.account-token": "Jeton Compte",
+  "token-type.app-token": "Jeton App",
+  "tokens.administrator-privileges-required": "Privilèges administrateur requis",
+  "tokens.button-create-token": "Créer jeton",
+  "tokens.create-tokens-description": "Créer des jetons pour accéder à l'",
+  "tokens.create-tokens-description-hubs-api": "API Hubs",
+  "tokens.no-access-description1": "Vous n'avez pas les privilèges suffisants pour créer des jetons API.",
+  "tokens.no-access-description2": "Si vous pensez que vous devriez avoir accès à cette page, veuillez demander des privillèges supplémentaires à votre administrateur Hubs.",
+  "tokens.no-tokens-created": "Aucun jeton créé.",
+  "tokens.revoke-token": "Revoquer",
+  "tokens.scopes": "Portées d'accès",
```

Comment:
"token-type.account-token": "Account Token",
  "token-type.app-token": "App Token",
  "tokens.administrator-privileges-required": "Administrator privileges required",
  "tokens.button-create-token": "Create token",
  "tokens.create-tokens-description": "Create tokens to access the",
  "tokens.create-tokens-description-hubs-api": "Hubs API",
  "tokens.no-access-description1": "You do not have sufficient privileges to create API tokens.",
  "tokens.no-access-description2": "If you believe you should have access to this page, please request privileges from your Hubs administrator.",
  "tokens.no-tokens-created": "No tokens created.",
  "tokens.revoke-token": "Revoke",
  "tokens.scopes": "Scopes",